Update Templating landing page for V8 (#2043)
umbraco/UmbracoDocs: Reference/Templating/index.md

---
versionFrom: 8.0.0
---

# Templating

_Templating in Umbraco consists of 3 larger concepts, namely Templates (Views), Partials (Partial Views) and Macros (Macro Partials). Templates are used for the HTML layout of your pages, partials can be included in your templates for shared functionality across different page templates and macros for reusable dynamic components that can be controlled by editors to embed functionality into the grid or rich text areas._

## Templating technology

In Umbraco V8 there is now only one type of templating technology:

> * MVC (views)

Author: Updated!

:::note
 The WebForms (masterpages) and dynamic razor approaches to templating are still available in Umbraco 7, but are now no longer supported in V8.
:::

#### [Working with MVC (views, razor, etc...)](Mvc/index.md)

> Describes how to work with MVC views, the razor syntax and APIs available as well details on how to create forms, step-by-step guides and other advanced techniques.

Reviewer: ```suggestion
Describes how to work with MVC views, the razor syntax and APIs available. It also describes how to create forms, has some step-by-step guides and other advanced techniques.
```
Found this sentence a bit confusing. Tried to make it less so, don't think it's great yet though 😁

Author: yes I didn't write that :-P
